.goreleaser.yml

```diff
@@ -85,35 +85,134 @@ docker_signs:
     args: ["sign", "--key=env://COSIGN_KEY", "--yes", "${artifact}"]
     artifacts: all
 
+# Build multiplatform images https://goreleaser.com/cookbooks/multi-platform-docker-images/
 dockers:
+  # control-plane
   - dockerfile: app/controlplane/Dockerfile.goreleaser
     ids:
       - control-plane
       - chainloop-plugin-discord-webhook
       - chainloop-plugin-smtp
       - chainloop-plugin-dependency-track
     image_templates:
-      - "ghcr.io/chainloop-dev/chainloop/control-plane:{{ .Tag }}"
```

Comment:
Can this potentially break the images in the charts? Because of the change in the tag basically

Reply:
Note that in the section below we are pushing this tag again, but as multiarch image instead. So basically nothing has changed.